lldb/test/API/tools/lldb-dap/console/TestDAP_console.py

```diff
@@ -104,3 +115,49 @@ def test_empty_escape_prefix(self):
             "Help can be invoked",
             command_escape_prefix="",
         )
+
+    @skipIfWindows
+    @skipIfRemote
+    def test_exit_status_message_sigterm(self):
+        source = "main.cpp"
+        program = self.getBuildArtifact("a.out")
+        self.build_and_launch(program, commandEscapePrefix="")
+        breakpoint1_line = line_number(source, "// breakpoint 1")
+        breakpoint_ids = self.set_source_breakpoints(source, [breakpoint1_line])
+        self.continue_to_breakpoints(breakpoint_ids)
+
+        # Kill lldb-server process.
+        process_name = (
+            "debugserver" if platform.platform() == "MacOS" else "lldb-server"
```

Comment:
On macOS, `platform.platform()` returns something like 'macOS-14.5-arm64-arm-64bit'. What you want to check here is `platform.system()` and check that it's "Darwin":

```
"debugserver" if platform.system() in ["Darwin"] else "lldb-server"
```